runtime/fswatch.vim

```diff
@@ -0,0 +1,15 @@
+" autocommands for starting filesystem based file watcher
```

Comment:
Thoughts on putting here vs. `runtime/plugin/fswatch.vim`?

Reply:
Ah that might actually work better then this. For now I can't get neovim to source files from `runtime`. I did read somewhere in the source that the plugin directory is automatically sourced. Probably in `main.c`?
I will move it there and see how that goes.

As for why I put it there, well,